.github/workflows/run-security-scans.yml

```diff
@@ -189,26 +205,68 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Set high and critical severity outputs
+        id: severity_check_outputs
+        run: |
+          severity_check_high=$(curl -s -H "Accept: application/vnd.github+json" -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" https://api.github.com/repos/${{ github.repository }}/code-scanning/alerts?ref=${{github.ref}} | jq '[.[] | select(.state == "open" and (.rule.security_severity_level == "high"))] | any') 
```

Comment:
Det er ikke noe garanti at man får ut noen alerts for aktuelle nivå da Github tjenesten er paginert.
Sjekken gir med andre ord false negatives. 

For mer informasjon se https://docs.github.com/en/rest/code-scanning?apiVersion=2022-11-28#list-code-scanning-alerts-for-a-repository 

Denne svakheten fantes i eksisterende versjon også, men en kan lett fikse dette her da man spør etter en og en severity om gangen :)

Reply:
Det er godt sett. Forhåpentligvis dukker det ikke opp superofte at folk har hatt 30+ security vulnerabilities, men det er absolutt noe å ta tak i. I fix :)